cmd/state-tray/internal/open/open_win.go

```diff
@@ -0,0 +1,23 @@
+//+build windows
+
+package open
+
+import (
+	"os/exec"
+
+	"github.com/ActiveState/cli/internal/locale"
+)
+
+// Prompt will open the a command prompt and execute the given command string
+func Prompt(command string) error {
+	// start will open an instance of the given executable. The first parameter
+	// of start is the title. The /k flag tells the cmd prompt to stay open
+	// after the command has completed execution.
+	cmd := exec.Command("cmd.exe", "/c", "start", "", "cmd.exe", "/k", command)
```

Comment:
What are the 4th and 5th arguments for? So the empty one and the second "cmd.exe".

Reply:
The empty string is the title. In this case, when a new command prompt is open that will be the string in the title bar. The second `cmd.exe` is the program to start. I've updated the comment slightly